gridsome/app/components/Image.js

```diff
@@ -24,6 +24,11 @@ export default {
     const attrs = data.attrs || {}
     const hook = data.hook || {}
     const res = []
+    
+    if (props.src === null) {
```

Comment:
I think you can remove these lines as the source of the bug might have been fixed in https://github.com/gridsome/gridsome/commit/918e76be556aece82ef50c23777f4acc2b2467e8